.github/workflows/tests_lkcuda_cpp.yml

```diff
@@ -190,7 +194,7 @@ jobs:
         run: |
           cd main
           cmake . -BBuild -G Ninja \
-            -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+            -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Could you add a Note for this change? Feel free to add a url to the failed test for the future reference. thanks!

Reply:
I've added the link in the PR description, i'll also link it here https://github.com/PennyLaneAI/pennylane-lightning/blob/e9a3b80823a2dc52f747ff20f91031512d0dbd63/pennylane_lightning/core/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_Param.cpp#L2062